app/change_set_persisters/change_set_persister/update_proxied_resource.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class ChangeSetPersister
+  class UpdateProxiedResource
```

Comment:
I initially expected this class to update the resource that `proxied_file_id` points to.  Maybe `UpdateProxyResource` or `CopyAttributesFromProxiedResource` would be clearer?